src/__internal__/form-field/form-field.spec.tsx

```diff
@@ -200,15 +207,20 @@ describe("FormField", () => {
   it.each(["error", "warning", "info"])(
     "validates the %s prop to throw an error if disabled is set as well",
     (validation) => {
-      jest.spyOn(global.console, "error").mockImplementation(() => {});
-      renderWithContext({ disabled: true, id: "foo", [validation]: true });
-      // eslint-disable-next-line no-console
-      expect(console.error).toHaveBeenCalledWith(
-        `Warning: Failed prop type: Prop \`${validation}\` cannot ` +
-          "be used in conjunction with `disabled`. Use `readOnly` if you require users to see validations with a " +
-          "non-interactive field\n    in FormField"
-      );
-      global.console.error.mockReset();
+      // Even though error is caught, it is still printed to console so mock console.error
```

Comment:
I seem to be having some issues with using `expectConsoleOutput` here. Since we are using `invariant` in our typescript refactors, the error is being thrown rather than just console logged, so we would need to use `toThrow()` in order to catch it correctly.

Reply:
That's fine 👍🏻